apps/web-giddh/src/app/new-inventory/component/adjust-inventory/adjust-inventory.component.html

```diff
@@ -34,6 +34,7 @@
                     </div>
                     <div class="col-xxl-3 mt-3 col-xl-3 col-lg-6 col-md-6">
                         <input-field
+                            [readonly]="referenceNumber"
```

Comment:
Use boolean value or this

```suggestion
                            [readonly]="!!referenceNumber"
```